sklearn/inspection/setup.py

```diff
@@ -1,18 +0,0 @@
-from numpy.distutils.misc_util import Configuration
-
-
-def configuration(parent_package="", top_path=None):
-    config = Configuration("inspection", parent_package, top_path)
-
-    config.add_subpackage("_plot")
-    config.add_subpackage("_plot.tests")
```

Comment:
I built the scikit-learn wheel locally using this branch and then unzipped it to a temp folder and I confirm those sub-packages are included automatically, presumably because of https://github.com/scikit-learn/scikit-learn/pull/24563#discussion_r1012962695.